lib/model.js

```diff
@@ -7,12 +7,11 @@
  * Module Dependencies.
  */
 
-var g = require('strong-globalize')();
```

Comment:
You can't remove this line. Other places in this file are using it and failing CI linter.